package.json

```diff
@@ -21,6 +21,7 @@
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.1.4",
     "react-select": "^2.4.2",
+    "react-select-plus": "^1.0.0-rc.3.patch12",
```

Comment:
How is this related to the ticket?

Reply:
I have updated it was a library I was using but I removed it.